statement.k

```diff
@@ -172,8 +172,8 @@ imports CONFIGURATION
 
 //Block
 
-         rule <k> {Ss:Statements} => Ss ~> Rho ...</k> <env> Rho </env> [structural]
-         rule <k> Rho:Map => . ... </k> <env> _ => Rho </env>           [structural]
+         rule <k> {Ss:Statements} => Ss ~> Rho ...</k> <env> Rho </env>  [structural]
+         rule <k> Rho => . ...</k> <env> _ => Rho </env>    [structural]        
```

Comment:
Any reason to remove the `Rho:Map`?
I think adding the sort information will be needed in the OCaml backend, so we may want to get into the habit to add it as we go (it should't be a problem for us now, but it will be a bit less work later).